test/plot-schema.json

```diff
@@ -10285,7 +10294,8 @@
       "values": [
        "auto",
        "linear",
-       "array"
+       "array",
+       "sync"
```

Comment:
You may possibly want to make this work minor grid lines i.e. `xaxis.minor` & `yxais.minor`.
Is that correct?

Reply:
Yes, correct.